classroom.rb

```diff
@@ -0,0 +1,51 @@
+require_relative './student'
+require_relative './person'
+require_relative './classroom'
+
+class Classroom
+  def initialize(label)
+    @label = label
+  end
+  attr_accessor :label
+
+  def add_student
+    new.Student(@label)
```

Comment:
- [ ] **Required:** Ensure that a student can be accessed/read from this class by using an attribute reader (`attr_reader`).